src/components/TopBar/TopBarMenu.vue

```diff
@@ -367,6 +368,13 @@ export default {
 			return this.canFullModerate && recordingEnabled
 		},
 
+		canConfigureBreakoutRooms() {
+			const breakoutRoomsEnabled = getCapabilities()?.spreed?.config?.call?.['breakout-rooms'] || false
+			return this.canFullModerate
+				&& breakoutRoomsEnabled
+				&& this.conversation.type === CONVERSATION.TYPE.GROUP
+		},
```

Comment:
```suggestion
		canConfigureBreakoutRooms() {
		  if (!this.conversation.type === CONVERSATION.TYPE.GROUP || !this.canFullModerate) {
		    return false
		  }
		  
			const breakoutRoomsEnabled = getCapabilities()?.spreed?.config?.call?.['breakout-rooms']
			return !!breakoutRoomsEnabled
		},
```

I'd prefer mentioned above order of checks, to reduce number of requests

Reply:
> to reduce number of requests

`getCapabilities` is not doing a request but reading from a local variable of the lib, but fine by me either way